site/app.yaml

```diff
@@ -10,3 +10,19 @@ handlers:
 - url: /(.*)
   static_files: _site/\1
   upload: _site/(.*)
+
+skip_files:
+- ^_apps/.*
+- ^_contributors/.*
+- ^_includes/.*
+- ^_layouts/.*
+- ^_sass/.*
+- ^assets/.*
+- ^node_modules/.*
+- ^_config\.yml$
+- ^Gemfile.*
+- ^package\.json$
+- ^sw-precache-config\.js$
+- ^(.*/)?.*\.md$
+- ^(.*/)?.*\.gpg$
+- ^(.*/)?\..*$
```

Comment:
Yep you're probably right, if we're not using those regexes here then it should be okay to not include them. I'm good either way :)